### PR TITLE
Add shaders

### PR DESCRIPTION
**Pull request by trocolit2**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,5 +2,9 @@
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 2.6)
 find_package(Rock)
+
 rock_init(vizkit3d_normal_depth_map 0.1)
 rock_standard_layout()
+
+install(DIRECTORY resources/shaders/
+    DESTINATION share/normal_depth_map/shaders)
```
Comment (doudou) [inline CMakeLists.txt]: use the full package name here (vizkit3d_normal_depth_map)

File: src/ImageViewerCaptureTool.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#include <iostream>
+#include <unistd.h>
+#include <osgDB/WriteFile>
+
+namespace vizkit3d_normal_depth_map {
+
+ImageViewerCaptureTool::ImageViewerCaptureTool(uint width, uint height) {
+
+    // initialize the hide viewer;
+    this->_viewer = new osgViewer::Viewer;
```
Comment (doudou) [inline src/ImageViewerCaptureTool.cpp]: unless you really want the "this->" attribute style, I would drop it... It's pretty uncommon (to my knowledge), horribly verbose (but that's a personal opinion) and definitely not a common style in Rock

Comment (doudou) [inline src/ImageViewerCaptureTool.cpp]: (But, note, it's you code, it's your style. This is a personal preference)

File: CMakeLists.txt (hunk 1 above)
Comment (trocolit2) [inline CMakeLists.txt]: OK! Correction done!

File: src/ImageViewerCaptureTool.cpp (hunk 2 above)
Comment (trocolit2) [inline src/ImageViewerCaptureTool.cpp]: HUM!! 
Thanks for feedback!
I will follow your suggestion.

**Pull request merged**